src/tools/schema2kotlin.ts

```diff
@@ -74,8 +80,8 @@ class KotlinGenerator implements ClassGenerator {
     this.fields.push(`var ${fixed}: ${type} = ${defaultVal}`);
 
     this.decode.push(`"${field}" -> {`,
-                     `  decoder.validate("${typeChar}")`,
-                     `  this.${fixed} = decoder.${decodeFn}`,
+                     `    decoder.validate("${typeChar}")`,
```

Comment:
2-space indent is google and kotlin's official style

Reply:
Huh. Good to know, thank you.

Reply:
I'll configure my IDE to match.